packages/bubble-dev/start-preset/src/build.ts

```diff
@@ -30,7 +30,7 @@ export const buildWeb = async (dir: string): Promise<StartPlugin<{}, {}>> => {
   return sequence(
```

Comment:
**this is a general thread**

regarding the screenshot, what actually we want to happened in the node folder?
we get from 3 files to two, so the node file is now..... ?

Reply:
Before it was treating files with `node` extension like a random helper file. Which means, it was just copied to build folder in every build.
I changed it to be treated as files with `native` extensions during build for native. It means that files with `node` extension are omitted in web and native build, but in node build, if there are two files with the same names, but one of them has `node` extension the one with `node` extension will be chosen to be put into build folder and renamed to remove the node extension.
The same happens in native build, if there is a file with `native` extension it will be taken over file with the same name and renamed to remove the extension.
I think that removing the extensions in build folders is there due to some typescript reasons, because theoretically you could just point in package.json to `'react-native': 'build/native/index.native.js'`, but I guess it's safer this way.
Does that answer your question?

Reply:
let me over complicated it for you 😂

Reply:
Yes! This is exactly how it works! Great visual explanation! 🥇 🏆